### DISCUSSION TITLE
How do i extract a letter from a string

### DISCUSSION THREAD

**Discussion by RGBprotons** [nesbox/TIC-80#1763] [General]
I frequently use Tic-80 as well as a site called scratch. In scratch there is a piece of code that reads "letter (index) of (string)" and it basically returns the character at (index) in (string). I'm look for the equivalent as in how do i plug in a string and a number and get the character used in that string at that place.

**Comment by Weeppiko** [comment]
This is not a TIC-80 specific question since it depends on which language you're using. But assuming you're using Lua, you can use `string.sub` to do it
